plugins/hypervisors/vmware/src/main/java/com/cloud/hypervisor/guru/VmwareVmImplementer.java

```diff
@@ -129,9 +129,8 @@ VirtualMachineTO implement(VirtualMachineProfile vm, VirtualMachineTO to, long c
                 }
             }
         } else {
-            // for user-VM, use E1000 as default
             if (nicDeviceType == null) {
-                details.put(VmDetailConstants.NIC_ADAPTER, VirtualEthernetCardType.E1000.toString());
```

Comment:
This looks good @GaOrtiga. Do we also need to use this setting in case of an Invalid NIC device type (in catch() in else case)